db/migrate/20200606210229_create_discount_items.rb

```diff
@@ -0,0 +1,9 @@
+class CreateDiscountItems < ActiveRecord::Migration[5.1]
```

Comment:
I had debated putting the merchant_id here too, but ultimately decided it was easily accessible through the item and the discount and therefore unnecessary.